### PR TITLE
Save image

### PR DESCRIPTION
**Pull request by zero77**
This PR is to save the current image to pictures directory.

### CODE REVIEW

File: styli.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-link="https://source.unsplash.com/random/"
+link="https://source.unsplash.com/random/1920x1080"
```
Comment (thevinter) [inline styli.sh:2]: This breaks [line 182](https://github.com/thevinter/styli.sh/blob/232f23ccb0dc9701372bc8aafd5b67399917c51e/styli.sh#L182)

**Review by thevinter — CHANGES_REQUESTED** [review]
Also would be cool if you could update the readme. Other than that thanks for the PR! Really appreciated :)

File: styli.sh (hunk 1 above)
Comment (zero77) [reply styli.sh]: I missed that but, it should be ok now

**Review by thevinter — APPROVED** [review]

**Comment by thevinter** [comment]
Thanks!

**Pull request merged**